platform/ATCmdParser.h

```diff
@@ -222,7 +222,7 @@ class ATCmdParser : private NonCopyable<ATCmdParser> {
      */
     bool recv(const char *response, ...) MBED_SCANF_METHOD(1, 2);
 
-    bool vrecv(const char *response, std::va_list args);
+    int vrecvscanf(const char *response, std::va_list args, bool isscnf);
```

Comment:
I think this now deserves its own doxygen documentation. `vrecv` was kind of wrapped by `recv` and `vscanf` was wrapped by `scanf`, so they never got their own doxygen. But now the new "Frankenstein's" `vrecvscanf` has an extra parameter (currently with a typo `isscnf`, but hopefully it would become something more descriptive, such as 'multiline' or sth similar) that is not documented anywhere and also its `int` return value fits more under the `scanf`'s doxygen?

Reply:
Done